api/types/crds/skupper_site_crd.yaml

```diff
@@ -81,57 +113,91 @@ spec:
                     properties:
                       host:
                         type: string
+                        description: |
+                          Hostname or IP address of the endpoint.
                       port:
                         type: string
+                        description: |
+                          Port number for the endpoint.
                       name:
                         type: string
+                        description: |
+                          Name of the endpoint.
                       group:
                         type: string
+                        description: |
+                          Group or category to which the endpoint belongs.
                 sitesInNetwork:
                   type: integer
+                  description: |
+                    The number of sites currently active in the network.
```

Comment:
currently active like in v1?

Reply:
I would say simply 'currently in the network' as the meaning of active here is not clear.